frontend/src/components/ExperimentCollection/ExperimentCollectionDashboard/ExperimentCollectionDashboard.tsx

```diff
@@ -39,7 +39,10 @@ export const ExperimentCollectionDashboard: React.FC<ExperimentCollectionDashboa
                             <Link to={"/" + exp.slug}>
                                 <ImageOrPlaceholder imagePath={exp.image} alt={exp.description} />
                                 <h3>{exp.name}</h3>
-                                <div role="status" className="counter">{exp.finished_session_count}</div>
+                                <div className="status-bar">
+                                    <span title={`Het "${exp.name}" experiment is ${exp.started_session_count} keer gestart`} role="status" className="counter">{exp.started_session_count}</span>
+                                    <span title={`Het "${exp.name}" experiment is ${exp.finished_session_count} keer voltooid`} role="status" className="counter">{exp.finished_session_count}</span>
+                                </div>
```

Comment:
We don't want text to be set directly through the frontend, to be able to customize / localize this via the backend. As we'll revert this change later anyway (see #931), this is okay for now though. Perhaps the text had better be in English though, since we're kind of custom building it for the Congo team now?

Reply:
As another comment - I believe the name of the experiment is displayed right above the status bar? So perhaps this could be just "Started {} times", "Finished {} times".